ts/features/bonus/cgn/types/DiscountBucketCodeResponse.ts

```diff
@@ -0,0 +1,38 @@
+import { DiscountBucketCode } from "../../../../../definitions/cgn/merchants/DiscountBucketCode";
+
+/**
+ * Type representing service preference successfully loaded
```

Comment:
```suggestion
 * Type representing service preference successfully loaded
```
maybe is a copy&paste error

Reply:
yes thx for noticing b199f44 fixes all the comments